Request parameter (#49)
tallence/core-redirects: docs/userguide.md

  - Only after 404: These redirects will only execute, if the source url results in a 404 Not found error. This can for
    example be used to redirect from missing job descriptions to a jobs overview page.
- Source: The source path for a redirect (which url to redirect)

> - Source Url Parameters: A url will only be redirected, if all parameters match.

Reviewer: ```suggestion
- Source Url Parameters: A url will only be redirected if all parameters match.
```

> - Source Url Type. This is only visible if configured:

Reviewer: ```suggestion
- Source Url Type. This is only visible if a user is a member of the required group:
```

  - Plain: Simple redirects, very fast, but only for a single source url

>   - Regular Expression: Defines a rule for several source urls to match. Slower, but more flexible.

Reviewer: ```suggestion
  - Regular Expression: Defines a rule for multiple source urls to match. Slower, but more flexible.
```

- Target Link: The target of the redirect (where to redirect to).
- Target Url Parameters: They will be appended to the redirect target url
- Target Url: The target of the redirect (where to redirect to). Should be used with care, because it might redirect to a non-existing page. The nice thing about linking contents is that the redirect target will exist as long as the redirect exists.
- Description :information_source: : Can be used to keep notes for this redirect. Also searchable.

Uploading Redirects
---------------------------

> You can upload multiple redirects at once via a CSV-File using the upload button (in the top left corner). The Uploader validates the input and lists the redirects which could not be imported.

Reviewer: ```suggestion
You can upload multiple redirects at once via a CSV file using the upload button (in the top left corner). The Uploader validates the input and lists the redirects which could not be imported.
```

> The expected CSV-Format is described in the upload dialog, an example-CSV can be found [here](./redirects.csv).

Reviewer: ```suggestion
The expected CSV format is described in the upload dialog, an example CSV can be found [here](./redirects.csv).
```
